packages/social-previews/types.d.ts

```diff
@@ -0,0 +1,21 @@
+declare module '@wordpress/notices' {
+	export type Status = 'error' | 'info' | 'success' | 'warning';
+}
+
+declare module '@wordpress/rich-text' {
```

Comment:
These two are a classic example of types we needed to specifically declare because those packages also attempted to declare the `@wordpress/data` module, which directly entered in conflict with the new package-provided `@wordpres/data` types. Ideally, each of those packages should provide types directly.

Reply:
Do you think it'd be worth trying to fix the DT defs? (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/wordpress__rich-text/index.d.ts)

Technically, I guess those are broken for everyone with new wordpress data versions.